{
  "cve": "CVE-2021-20219",
  "mitre": {
    "cpes": [],
    "created": "2021-03-23T16:34:00+00:00",
    "description": "A denial of service vulnerability was found in n_tty_receive_char_special in drivers/tty/n_tty.c of the Linux kernel. In this flaw a local attacker with a normal user privilege could delay the loop (due to a changing ldata->read_head, and a missing sanity check) and cause a threat to the system availability.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2021/20xxx/CVE-2021-20219.json",
    "references": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=1923738"
    ],
    "title": null,
    "updated": "2024-08-03T17:30:07.576000+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-697"
    ]
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*"
    ],
    "created": "2021-03-23T17:15:13.607000+00:00",
    "description": "A denial of service vulnerability was found in n_tty_receive_char_special in drivers/tty/n_tty.c of the Linux kernel. In this flaw a local attacker with a normal user privilege could delay the loop (due to a changing ldata->read_head, and a missing sanity check) and cause a threat to the system availability.",
    "metrics": {
      "cvssV2_0": {
        "score": 2.1,
        "vector": "AV:L/AC:L/Au:N/C:N/I:N/A:P"
      },
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 5.5,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2021/CVE-2021-20219.json",
    "references": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=1923738"
    ],
    "title": null,
    "updated": "2021-03-26T19:48:33.537000+00:00",
    "vendors": [
      "linux",
      "linux$PRODUCT$linux_kernel"
    ],
    "weaknesses": [
      "CWE-697"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd"
      ]
    },
    "created": {
      "data": "2018-05-26T03:30:00+00:00",
      "provider": "redhat"
    },
    "description": {
      "data": "A denial of service vulnerability was found in n_tty_receive_char_special in drivers/tty/n_tty.c of the Linux kernel. In this flaw a local attacker with a normal user privilege could delay the loop (due to a changing ldata->read_head, and a missing sanity check) and cause a threat to the system availability.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {
          "score": 2.1,
          "vector": "AV:L/AC:L/Au:N/C:N/I:N/A:P"
        },
        "provider": "nvd"
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 5.5,
          "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
        },
        "provider": "nvd"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": "Moderate",
        "provider": "redhat"
      }
    },
    "references": {
      "data": [
        "https://bugzilla.redhat.com/show_bug.cgi?id=1923738",
        "https://nvd.nist.gov/vuln/detail/CVE-2021-20219",
        "https://www.cve.org/CVERecord?id=CVE-2021-20219"
      ],
      "providers": [
        "mitre",
        "nvd",
        "redhat"
      ]
    },
    "title": {
      "data": "kernel: improper synchronization in flush_to_ldisc() can lead to DoS",
      "provider": "nvd"
    },
    "updated": {
      "data": "2024-08-03T17:30:07.576000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "linux",
        "linux$PRODUCT$linux_kernel"
      ],
      "providers": [
        "nvd"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-697"
      ],
      "providers": [
        "mitre",
        "nvd",
        "redhat"
      ]
    }
  },
  "redhat": {
    "cpes": [],
    "created": "2018-05-26T03:30:00+00:00",
    "description": "A denial of service vulnerability was found in n_tty_receive_char_special in drivers/tty/n_tty.c of the Linux kernel. In this flaw a local attacker with a normal user privilege could delay the loop (due to a changing ldata->read_head, and a missing sanity check) and cause a threat to the system availability.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 5.5,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
      },
      "threat_severity": "Moderate"
    },
    "redhat_repo_path": "2021/CVE-2021-20219.json",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2021-20219",
      "https://www.cve.org/CVERecord?id=CVE-2021-20219"
    ],
    "title": "kernel: improper synchronization in flush_to_ldisc() can lead to DoS",
    "updated": "2018-05-26T03:30:00+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-697"
    ]
  }
}